Service/src/routes/transaction.py

```diff
@@ -90,7 +90,7 @@ def sell(sharePrice, requestBody):
    g.cursor.execute("INSERT INTO Transaction" +
       "(sharePrice, shareCount, action, portfolioId, ticker, leagueId) " +
       "VALUES (%s, %s, %s, %s, %s, %s)",
-      [sharePrice, requestBody['shareCount'], 0, requestBody['portfolioId'], requestBody['ticker'], portfolio['leagueId']])
+      [sharePrice, requestBody['shareCount'], "SELL", requestBody['portfolioId'], requestBody['ticker'], portfolio['leagueId']])
```

Comment:
why is this change necessary?

Reply:
Clarity, why call the action "0" instead of "SELL"

Reply:
ok

Reply:
keep in mind there are several ways to sell in a stock market, at present we only support "market sell" so "SELL" is definitely ambigious

that's probably why we did the numeric before